initializr-service/src/main/resources/application.yml

```diff
@@ -1292,6 +1300,38 @@ initializr:
           id: remote-shell
           description: CRaSH shell integration
           versionRange: "[1.0.0.RELEASE,1.5.0.RC1)"
+    - name: Spring Cloud GCP
+      bom: spring-cloud-gcp
+      versionRange: 2.0.0.RC1
+      content:
+        - name: GCP Support
+          id: cloud-gcp
+          description: Spring Boot autoconfiguration to use GCP services through Spring Cloud
+          groupId: org.springframework.cloud
+          artifactId: spring-cloud-gcp-starter
+          links:
+            - rel: reference
+              href: https://docs.spring.io/spring-cloud-gcp/docs/1.0.0.M2/reference/htmlsingle/
+              description: Reference doc
+        - name: GCP Messaging
+          id: cloud-gcp-pubsub
+          description: Spring Boot autoconfiguration for Spring Cloud GCP Pub/Sub
+          groupId: org.springframework.cloud
+          artifactId: spring-cloud-gcp-starter-pubsub
+          links:
+            - rel: reference
+              href: https://docs.spring.io/spring-cloud-gcp/docs/1.0.0.M2/reference/htmlsingle/
+              description: Reference document
+        - name: GCP Storage
+          id: cloud-gcp-storage
+          description: Spring Boot autoconfiguration for Spring Cloud GCP Storage
+          groupId: org.springframework.cloud
+          artifactId: spring-cloud-gcp-starter-storage
+          links:
+            - rel: reference
+              href: https://docs.spring.io/spring-cloud-gcp/docs/1.0.0.M2/reference/htmlsingle/
```

Comment:
Can we have a link that is dedicated on messaging please? a general link for a dedicated entry doesn't sound very useful.